include/swift/ClangImporter/CXXMethodBridging.h

```diff
@@ -62,16 +62,21 @@ struct CXXMethodBridging {
   }
 
   NameKind classifyNameKind() {
-    bool allLower = llvm::all_of(getClangName(), islower);
+    bool hasUpper = false;
+    if (std::count_if(getClangName().begin(), getClangName().end(), [](unsigned char ch) { return std::isupper(ch); }) >= 1) {
```

Comment:
I don't see why this is necessary. What's the error for `llvm::all_of(getClangName(), [](unsigned char ch) { return std::isupper(ch); })`?

Feel free to run the linux bots as much as you want, if you don't have a VM or something local. Or ask someone to run them for you.

Reply:
Ahh yeah, when i tried that all the tests for uppercase failed, looks like it isnt catching. I will try to simplify the expression more

Reply:
looks like ther\e was an issue picking up the upper case when using the `llvm::all_of`. But i think this should work now, instead of calling the `llvm` method, just using the `std` call for `any_of`